snowflake.yml

```diff
@@ -0,0 +1,10 @@
+definition_version: 1
+native_app:
+  name: NATIVE_APP_QUICKSTART_PACKAGE
+  source_stage: app_src.stage
+  artifacts:
+    - src: app/src/*
+      dest: ./
+  package:
+    scripts:
+      - app/src/scripts/setup-package-script.sql
```

Comment:
The package script should not live in a file that makes its way to the stage, can you move it out of `app/src`?